azure-pipelines/ui-automation/msal-nightly.yml

```diff
@@ -0,0 +1,166 @@
+# run broker UI automation testcases
```

Comment:
nit : should this be run msal automation testcases?

Reply:
yes

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1617/commits/35c3e05a87b8a8e00af8b61d6bbfc45656d0f2a2